plugin/trino-blackhole/src/main/java/io/trino/plugin/blackhole/BlackHoleMetadata.java

```diff
@@ -24,6 +24,7 @@
 import io.trino.spi.connector.ColumnMetadata;
 import io.trino.spi.connector.ConnectorAnalyzeMetadata;
 import io.trino.spi.connector.ConnectorInsertTableHandle;
+import io.trino.spi.connector.ConnectorMaterializedViewDefinition;
```

Comment:
Can we add some tests related to MVs in `TestBlackHoleSmoke` and can we also open a GH issue to track the missing test coverage for BlackHole connector ?

Reply:
testa added